plasmapy/particles/tests/test_atomic.py

```diff
@@ -781,3 +783,8 @@ def test_missing_atomic_data(self):
 @pytest.mark.parametrize("particle, electron", str_electron_table)
 def test_is_electron(particle, electron):
     assert _is_electron(particle) == electron
+
+
+def test_integer_charge():
+    with pytest.warns(PlasmaPyFutureWarning):
+        assert integer_charge("Fe 20+") == 20
```

Comment:
If you want to be thorough, then it would be worthwhile to do a `mock` test here to ensure `integer_charge` calls `charge_number`.   It's a bit overkill, but it does enforce the connection between the two.

Reply:
It's probably more effort than is worth it for a deprecated feature, though this idea is worth keeping in mind for future features that we intend to keep around for a while.